src/hotspot/share/gc/shenandoah/shenandoahFreeSet.cpp

```diff
@@ -1130,6 +1130,16 @@ void ShenandoahFreeSet::reserve_regions(size_t to_reserve) {
   }
 }
 
+void ShenandoahFreeSet::log_status_under_lock() {
+  // Must not be heap locked, it acquires heap lock only when log is enabled
+  shenandoah_assert_not_heaplocked();
+  if (LogTarget(Info, gc, free)::is_enabled()
+      DEBUG_ONLY(|| LogTarget(Debug, gc, free)::is_enabled())) {
+    ShenandoahHeapLocker locker(_heap->lock());
+    log_status();
```

Comment:
If you do the changes, fix up the indenting here as well. I think one space is missing here in 2-space indent.

Reply:
I thinks is the line 1140 having indent issue, fixed.